hummingbird/ml/operator_converters/_gbdt_commons.py

```diff
@@ -41,6 +41,10 @@ def convert_gbdt_classifier_common(tree_infos, get_tree_parameters, n_features,
         n_classes -= 1
     if classes is None:
         classes = [i for i in range(n_classes)]
+    reorder_trees = True
+    if constants.REORDER_TREES in extra_config:
```

Comment:
I'm confused about why this is in this PR.  Does it relate to onnx, or part of some other refactoring?

Reply:
yes this is something needed for onnx and so that we can hook into the gbdt code.